packages/kit/types/index.d.ts

```diff
@@ -1911,8 +1912,6 @@ declare module '$app/navigation' {
 	 *
 	 * @param url Where to navigate to. Note that if you've set [`config.kit.paths.base`](https://kit.svelte.dev/docs/configuration#paths) and the URL is root-relative, you need to prepend the base path if you want to navigate within the app.
 	 * @param {Object} opts Options related to the navigation
-	 * @param invalidateAll If `true`, all `load` functions of the page will be rerun. See https://kit.svelte.dev/docs/load#rerunning-load-functions for more info on invalidation.
-	 * @param opts.state The state of the new/updated history entry
```

Comment:
it looks like these were previously present because `opts.invalidateAll` was missing `opts.`, but it's still wrong now — all the `@param {boolean} [opts.foo]` annotations here... https://github.com/sveltejs/kit/blob/ded16305b4c180a697c457ca1b2bec249b4b38fb/packages/kit/src/runtime/app/navigation.js#L23-L26 ...have been stripped out

Reply:
Tbh I think these only exist for the generated docs, from the jsdoc perspective these `@param` declarations are not really valid syntax (or is it?)

Reply:
Huh, yeah maybe. It would be nice to have the documentation present when you're passing in the options. Not sure how best to have and also eat our cake

Reply:
going to leave this as a problem for future us, don't want to get bogged down in it now